app/src/main/java/com/cornellappdev/transit/ui/components/AddFavoritesSearchSheet.kt

```diff
@@ -92,32 +107,37 @@ fun AddFavoritesSearchSheet(
 
             }
 
-            SearchTextField(
-                value = searchBarValue,
-                setValue = { s -> routeViewModel.onQueryChange(s) },
-                placeholderText = "Search",
-                singleLine = true,
-                modifier = Modifier.padding(
-                    start = 16.dp,
-                    end = 16.dp,
-                    top = 7.dp,
-                    bottom = 11.dp
+            DockedSearchBar(
+                query = addSearchBarValue,
+                onQueryChange = { s -> homeViewModel.onAddQueryChange(s) },
+                onSearch = { addSearchActive = false; homeViewModel.onSearch(it) },
+                active = addSearchActive,
+                onActiveChange = { b -> addSearchActive = b },
+                shape = RoundedCornerShape(size = 8.dp),
+                colors = SearchBarDefaults.colors(
+                    containerColor = Color.White,
+                    dividerColor = DividerGrey,
                 ),
-                height = 50.dp,
-                prefix = {
-                    Icon(
-                        Icons.Outlined.Search,
-                        "Search",
-                        //TODO: replace with actual search function
-                        modifier = Modifier.clickable { routeViewModel.onQueryChange("") })
-                },
-                suffix = {
-                    Icon(
-                        Icons.Outlined.Clear,
-                        "Clear",
-                        modifier = Modifier.clickable { routeViewModel.onQueryChange("") })
+                leadingIcon = { Icon(Icons.Outlined.Search, "Search") },
+                trailingIcon = { Icon(Icons.Outlined.Info, "Info") },
+                placeholder = { Text(text = "Search for a stop to add") }
+
```

Comment:
Delete empty line here!!

There's a good amount of cases like this in this file, actually. Could you on your next update to this PR just like comb through the file really quickly and remove any other weird random whitespace new lines? There are some whitespaces that serve a good purpose in that they separate important distinct parts of code, but then there's also stuff like this for example:

![image](https://github.com/user-attachments/assets/aa5acaab-3f62-4180-8673-d6ca2b55dfdb)

Reply:
I suppose there are some instances of this also in `HomeScreen.kt` if you don't mind looking there too :+1:

Reply:
Fixed `HomeScreen.kt` asw!